### ISSUE TITLE
How do I ensure the connector only inserts a nested object

### ISSUE THREAD

**Issue by JordanPawlett** [confluentinc/kafka-connect-elasticsearch#578]
I'm looking to only insert part of the message consumed from my topic. Our payloads contain protocol specific information, and the nested attribute is all we care about getting into Elasticsearch.

Currently we have a microservice that consumes from a topic and inserts into ES via the REST API. It would make more sense to move to a connector.

Example `value` of a message on our topic.
```json
{
  "ver": 4,
  "sender": "foo-bar",
  // actual payload we want inserting
  "data": {
    "uid": "foo-uid=bar",
    "name": "john",
    "timestamp": 145345345345
  }
}
```
I would like to only insert payload.data into the elasticsearch index.

Sorry if this has been answered before. I'm new to connectors, I've looked through the documentation and thought transforms may be the answer, but non of them seem relevant.

**Comment by lovedworking** [comment]
do you have the answer?

**Comment by JordanPawlett** [comment]
> do you have the answer?

No, it would be great if someone could shed some light on this!

**Comment by lovedworking** [comment]
> Try kafka connector's SMT. For your case i think it should be this one. https://docs.confluent.io/platform/current/connect/transforms/replacefield.html

ok i'll try it
